practice/2_python_part_2/task_read_write_2.py

```diff
@@ -22,3 +22,25 @@ def generate_words(n=20):
         words.append(word)
 
     return words
+
+def write_to_file():
+    index = 1
+    table = generate_words()
+    while(True):
+        if not exists("files/file_" + str(index) + ".txt"):
+            filepath = "files/file_" + str(index) + ".txt"
+            f = open(filepath, "x")
+            for item in table:
+                f.write(item+'\n')
```

Comment:
writing to file every iteration increasing number of I/O operation. So.. is not optimal way. You could prepare a string with needed separator using the join function and then write it once.